src/dakara_base/utils.py

```diff
@@ -129,3 +129,18 @@ def create_url(
 
 class URLParameterError(DakaraError, ValueError):
     """Error raised when server parameters are unproperly set."""
+
+
+def strtobool(input_value: str, default: bool = False) -> bool:
+    """
+    Convert user input to a boolean
+    """
```

Comment:
Please format the docstring as the others in the project (using the [Google format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)).

Reply:
should be fine with 35d2f690ca4efd918bda4fad67dddc2e6fa7fd7e